src/Microsoft.Diagnostics.Runtime/Microsoft.Diagnostics.Runtime.csproj

```diff
@@ -34,7 +34,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
-    <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
+    <PackageReference Include="System.Collections.Immutable" Version="6.0.0" />
```

Comment:
Why is this downgraded?  With 4.0 I have been pushing versions up to their newest release, as long as those releases still support 6.0 (which is the most recent in support).

Reply:
I could not get SOS to run with this change (and SOS's reference) with 8.0.0 hosted under 6.0. I'll try again to make sure I built everything correctly.

Reply:
Using 8.0 to host under windbg works but 6.0 and 7.0 don't.

Reply:
Ok, we can downgrade for now, but we can't remain pinned on 6.0.0 forever.  We'll have to figure out why `8.0.0` says it supports 6.0 runtimes (see https://www.nuget.org/packages/System.Collections.Immutable/) but doens't work when loaded there.

Reply:
There might something I can do in SOS's hosting code that builds the "TPA". I'll have to investigate.